components/layout_2020/flexbox/layout.rs

```diff
@@ -104,6 +104,10 @@ struct FlexLineLayoutResult {
     item_fragments: Vec<(BoxFragment, PositioningContext)>, // One per flex item, in the given order
 }
 
+fn css_pixel_length_to_au(length: CSSPixelLength) -> Au {
+    Au::from_f32_px(length.px())
+}
```

Comment:
I'm fairly sure there is already a conversion implementation of Into or From so you shouldn't need to add this helper and can likely just do length_value.into().